sites/funmooc/src/frontend/scss/settings/_variables.scss

```diff
@@ -158,6 +158,7 @@ $r-footer-logo-width-lg: 12.125rem;
 // full width like common blocks. Usefull if you plan to remove course run from
 // template
 $r-course-aside: 25%;
+$r-course-subheader-aside: 25%;
```

Comment:
Didn't you want to increase the aside column width ? On the demo website, the value is `35%`.

Reply:
hmm let's keep it for another PR when we will activate ecommerce...